azure-communication-ui/azure-communication-ui-demo-app/src/main/java/com/azure/android/communication/ui/callingcompositedemoapp/CallLauncherActivity.kt

```diff
@@ -182,20 +228,58 @@ class CallLauncherActivity : AppCompatActivity() {
         }
     }
 
-    fun showAlert(message: String) {
-        runOnUiThread {
-            val builder = AlertDialog.Builder(this).apply {
-                setMessage(message)
-                setTitle("Alert")
-                setPositiveButton("OK") { _, _ ->
-                }
-            }
-            builder.show()
+    private fun saveState(outState: Bundle?) {
+        outState?.putBoolean(isTokenFunctionOptionSelected, callLauncherViewModel.isTokenFunctionOptionSelected)
```

Comment:
Instead of bundle, consider shared preferences so this will persist between sessions.

Reply:
I think we can modify this as part of separate Pr as this code is not touched in this PR.